docs/sources/mimir/operators-guide/run-production-environment/scaling-out.md

```diff
@@ -20,6 +20,16 @@ However, be careful when scaling down some of the stateful components as these a
 When running Grafana Mimir in monolithic mode, you can safely scale up to any number of instances.
 To scale down the Grafana Mimir cluster, see [Scaling down ingesters](#scaling-down-ingesters).
 
+## Read-Write mode
+
+When running Grafana Mimir in Read-Write mode, you can safely scale up any of the services.
```

Comment:
Can we use a different naming than "service"? Service can be easily misunderstood with K8S Service. What if we just call them components, like in microservices?